python/stemwords.py

```diff
@@ -0,0 +1,103 @@
+import sys
+import re
+import codecs
+import snowballstemmer
+
+def usage():
+    print('''usage: %s [-l <language>] [-i <input file>] [-o <output file>] [-c <character encoding>] [-p[2]] [-h]
+
+The input file consists of a list of words to be stemmed, one per
+line. Words should be in lower case, but (for English) A-Z letters
+are mapped to their a-z equivalents anyway. If omitted, stdin is
```

Comment:
I don't see any code below to lowercase the input word (as there is in the C version).